### PR TITLE
scylla_node: add run_scylla_sstable()

### PR DESCRIPTION
**Pull request by denesb**
Convenience method allowing the invocation of the scylla-sstable tool, built into the scylla sstable. The method uses the same mechanism for selecting which sstable to work on, as Node.run_sstable2json(). But other than that, there are notable differences, run_scylla_sstable():
* Allows for accessing all scylla-sstable operations, not just dump-data (the scylla-sstable equivalent to sstable2json).
* Allows direct pass-through of command-line arguments to the underlying scylla-sstable invokation. This ensures maximum forward-compatibility, without the need to patch this method each time the underlying code's capabilities change.
* Has a batch mode, which allows for passing all sstables at once to scylla-sstable (which accepts 1 or more sstable args).
* It looses the `enumerate_keys` parameter.
* Returns the (stdout, stderr) for each underlying invocation, instead of writing the output to a file.

### CODE REVIEW

File: ccmlib/scylla_node.py [hunk 1]
```diff
@@ -1348,6 +1348,45 @@ def gnutls_config_file(self):
...
+        if additional_args is None:
+            additional_args = []
+        scylla_path = common.join_bin(self.get_path(), BIN_DIR, 'scylla')
+        env = self.get_env()
+        sstables = super()._Node__gather_sstables(datafiles, keyspace, column_families)
+        ret = {}
+
+        def do_invoke(sstables):
+            common_args = [scylla_path, "sstable", command] + additional_args
+            res = subprocess.run(common_args + sstables, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
```
Comment (fruch) [inline ccmlib/scylla_node.py:1378]: Add

text=True

And you can remove the decode calls from below

File: ccmlib/scylla_node.py [hunk 2]
```diff
@@ -1348,6 +1348,45 @@ def gnutls_config_file(self):
...
+
+        Returns: map: {sstable: (stdout, stderr)} of all invokations. When batch == True, a single entry will be present, with empty key.
+
+        Raises: subprocess.CalledProcessError if scylla-sstable returns a non-zero exit code.
+        """
+        if additional_args is None:
+            additional_args = []
+        scylla_path = common.join_bin(self.get_path(), BIN_DIR, 'scylla')
+        env = self.get_env()
+        sstables = super()._Node__gather_sstables(datafiles, keyspace, column_families)
```
Comment (fruch) [inline ccmlib/scylla_node.py:1373]: Why use super and not self ?

File: ccmlib/scylla_node.py (hunk 1 above)
Comment (denesb) [reply ccmlib/scylla_node.py:1378]: Neat, fixed in v2.

File: ccmlib/scylla_node.py (hunk 2 above)
Comment (denesb) [reply ccmlib/scylla_node.py:1373]: It took me a while to figure out how to call the method with `__` prefix. I guess `super()` is a leftover of things I tried. Removed in v2.

**Comment by denesb** [comment]
V2:
* Add `text=True` to `subprocess.run()` args.
* Use `self` instead of `super()`.

**Pull request merged**